src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -3280,52 +3283,64 @@ namespace SurfaceGeometry {
 
         // REFERENCES:
         // Detached Shading Surface Definition(s)
-        // Surface:Shading:Detached:Fixed,
```

Comment:
These types of comments can be deleted entirely.

Reply:
Code cleaned up along the way.